src/main/scala/com/exasol/cloudetl/kafka/deserialization/JsonDeserialization.scala

```diff
@@ -20,8 +21,12 @@ object JsonDeserialization extends RecordDeserialization {
       }
     ) {
       throw new KafkaConnectorException(
-        "Referencing all fields with key.* or value.* is not supported for json " +
-          "as the order is not deterministic."
+        ExaError
+          .messageBuilder("E-KCE-16")
+          .message("Referencing all fields with key.* or value.* is not supported ")
+          .message("for JSON as the order is not deterministic.")
```

Comment:
do it in once massage and concatenate with `+`

Reply:
as long as you only concatenate constants that's ok for the error-code-crawler and it expresses better the semantic that it's one message

Reply:
Okay, I switched to this couple more reverting them also.